### PR TITLE
docs: Remove outdated link in doc/fuzzing.md

### LINKED ISSUE #15003: [doc] fuzzing.md link is outdated
I was looking at the fuzzing.md docs to learn how Bitcoin Core does fuzzing (I am working on fuzzing another project) and I noticed one of the links is outdated: http://strateman.ninja/fuzzing.tar.xz.
The website is unavailable.

### PR DESCRIPTION
**Pull request by RandyMcMillan**
Fixes #15003

The change does not change fundamental bitcoin-core functionality and is limited to documentation.

### CODE REVIEW

File: doc/fuzzing.md [hunk 1]
```diff
@@ -56,7 +56,6 @@ AFLOUT=$PWD/outputs
 Example inputs are available from:
 
 - https://download.visucore.com/bitcoin/bitcoin_fuzzy_in.tar.xz
```
Comment (MarcoFalke) [inline doc/fuzzing.md]: Could add the hash of the file and a mirror url to avoid this happening in the future?

**Comment by laanwj** [comment]
Why didn't anyone tag me here.
I'll put the file back.

**Comment by laanwj** [comment]
@pstratem can you have a look?

**Comment by laanwj** [comment]
Closing this as #15043 is going to make the entire fuzzing corpus in need of re-generation, so there's no need to remove individual links.

**Pull request closed without merging**